BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsDatabase.java

```diff
@@ -1,219 +1,30 @@
 package com.braintreepayments.api;
 
-import android.content.ContentValues;
 import android.content.Context;
-import android.database.Cursor;
-import android.database.sqlite.SQLiteDatabase;
-import android.database.sqlite.SQLiteException;
-import android.database.sqlite.SQLiteOpenHelper;
-import android.os.AsyncTask;
 
-import androidx.annotation.VisibleForTesting;
+import androidx.room.Database;
+import androidx.room.Room;
+import androidx.room.RoomDatabase;
 
-import org.json.JSONException;
-import org.json.JSONObject;
+@Database(entities = {AnalyticsEvent.class}, version = 1)
+abstract class AnalyticsDatabase extends RoomDatabase {
 
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
-
-class AnalyticsDatabase extends SQLiteOpenHelper {
-
-    private static final String DATABASE_NAME = "braintree-analytics.db";
-    private static final int DATABASE_VERSION = 1;
-    private static final String TABLE_NAME = "analytics";
-
-    static final String ID = "_id";
-    static final String EVENT = "event";
-    static final String TIMESTAMP = "timestamp";
-    static final String META_JSON = "meta_json";
+    abstract AnalyticsEventDao analyticsEventDao();
 
     private static volatile AnalyticsDatabase INSTANCE;
 
-    static AnalyticsDatabase getInstance(Context context) {
+    static AnalyticsDatabase getDatabase(final Context context) {
```

Comment:
Nitpick - I kind of prefer `getInstance()` since we are returning a single instance here and we use that pattern elsewhere.

Reply:
Yeah this was straight-up copy-paste from the Google tutorial lol.